packages/generate-persisted-query-manifest/src/index.ts

```diff
@@ -19,41 +19,64 @@ import { first, sortBy } from "lodash";
 import { createHash } from "node:crypto";
 import { readFileSync } from "node:fs";
 
-export interface GeneratePersistedQueryManifestOptions {
-  files?: string[];
+type RequireKeys<T, K extends keyof T> = Required<Pick<T, K>> & Omit<T, K>;
 
-  globInclude?: string[];
-  globExclude?: string[];
+export interface PersistedQueryManifestConfig {
+  /**
+   * Paths to your GraphQL documents: queries, mutations, subscriptions, and fragments.
+   */
+  documents?: string | string[];
```

Comment:
Should we just endeavor to be fully codegen-compatible (since many folks might use both tools) and let you specify your ignores here too with `!` like codegen does? Looks like they use `globby` under the hood. (Or maybe we just should use `@graphql-tools/code-file-loader` which is one layer of wrapping up from the graphql-tag-pluck package we're using now?)

Reply:
I considered that as well and would be happy to reduce the number of options here if we like this better. I opted to maintain them separate, but only because you had created them separate initially 😆.

Reply:
I guess my main reaction was that it's weird to have "patterns" in one name and not the other; a nice way to resolve that is just "do the same exact thing as graphql-code-generator", and it looks like that might be relatively straightforward by switching to `globby`.

Reply:
Yep for sure. Naming is hard and I couldn't think of anything better since `documentsIgnore` sounded weird and was on the fence of whether `ignoredDocuments` would make sense to people.

I'm all for simplifying to match graphql code generator 100% though so I'll make this change.